setup.py

```diff
@@ -13,7 +13,7 @@
 	packages = find_packages(exclude=['tests', 'tests.*']),
 	install_requires = [
 		# Also update requirements/base.txt when you make changes here.
-		'selenium==3.141.0'
+		'selenium>=3.141.0'
```

Comment:
This permits Selenium 3.141.0. Will your `By.` changes still work with Selenium 3?

Reply:
I suppose that to support both major release, a version check must be added.